tests/monster_test_generated.js

```diff
@@ -0,0 +1,936 @@
+// automatically generated by the FlatBuffers compiler, do not modify
+
+/**
+ * @const
+*/
+var MyGame = MyGame || {};
+
+/**
+ * @const
+*/
+MyGame.Example = MyGame.Example || {};
+
+/**
+ * @const
+*/
+MyGame.OtherNameSpace = MyGame.OtherNameSpace || {};
+
+/**
+ * @enum
+ */
+MyGame.Example.Color = {
+  Red: 1,
+  Green: 2,
+  Blue: 8
+};
+
+/**
+ * @enum
+ */
+MyGame.Example.Any = {
+  NONE: 0,
+  Monster: 1,
+  TestSimpleTableWithEnum: 2
+};
+
+/**
+ * @constructor
+ */
+MyGame.Example.Test = function() {
+  /**
```

Comment:
The type checking that Google Closure Compiler provides was very useful for me in porting the library to JavaScript and having extra confidence that things were wired up correctly. It caught quite a few issues around boolean <=> integer conversions, for example. That said, now that the library is complete I guess it's ok to remove them if you want. It's sort of like removing tests: the tests have already provided their usefulness and it's fine to remove them as long as you don't need to make modifications later on. It's up to you, so I'll wait to hear what you decide.